apps/fm/js/fm.js

```diff
@@ -918,11 +918,18 @@ function init() {
   );
 }
 
-window.addEventListener('load', function(e) {
-  AirplaneModeHelper.ready(function() {
-    airplaneModeEnabled = AirplaneModeHelper.getStatus() == 'enabled';
-    init();
-
+navigator.mozL10n.once(function(e) {
```

Comment:
what if we already fired localized before the listener is set? (this script is loaded after l10n.js)